src/datasource.ts

```diff
@@ -179,7 +179,6 @@ export class BigQueryDatasource {
   private queryModel: BigQueryQuery;
   private readonly baseUrl: string;
   private readonly url: string;
-  private mixpanel;
```

Comment:
I get that we don't want to allow tracking. This will make it harder to track upstream or to push PRs upstream. Is there a problem with requiring it and setting `jsonData.sendUsageData` to `false`?

Reply:
This ended up not being an issue 🙈 I must have accidentally had the tracking still enabled.